guides/doc-Managing_Content/master.adoc

```diff
@@ -1,6 +1,6 @@
 include::common/attributes.adoc[]
 include::common/header.adoc[]
-:context: content-management
+:context: :content-management:
```

Comment:
This looks a bit off to me. Is there any reason for this change? AFAIK this would result in broken "{...DocURL}" anchors that previously used "_content-management" as rendered "_{context}".

Even if it does not change anything, please revert it unless there's a specific reason for this change. Then, I'd need to investigate some more about possible sideeffects.

Reply:
This wasn't intended. See below.